cmd/utils/flags.go

```diff
@@ -731,6 +736,7 @@ func RegisterEthService(ctx *cli.Context, stack *node.Node, extra []byte) {
 		blockTimeMillis := ctx.GlobalInt(RaftBlockTimeFlag.Name)
 		datadir := ctx.GlobalString(DataDirFlag.Name)
 		joinExistingId := ctx.GlobalInt(RaftJoinExistingFlag.Name)
+		raftPort := uint16(ctx.GlobalInt(RaftPortFlag.Name))
```

Comment:
Similarly, this silently overflows if a value above `2^16` is provided. I probably wouldn't care except the default of `50400` is most of the way there.